lib/src/airship_flutter.dart

```diff
@@ -310,4 +311,30 @@ class Airship {
   static Future<bool?> setPushTokenRegistrationEnabled(bool enabled) async {
     return await _channel.invokeMethod('setPushTokenRegistrationEnabled', enabled);
   }
+
+  static Future<bool> isAutoBadgeEnabled() async {
+    var isAutoBadgeEnabled = false;
+    if (Platform.isIOS) {
+      isAutoBadgeEnabled = await _channel.invokeMethod('isAutoBadgeEnabled');
+    }
+    return isAutoBadgeEnabled;
+  }
+
+  static Future<void> setAutoBadgeEnabled(bool enabled) async {
+    if (Platform.isIOS) {
```

Comment:
it looks like these set methods will never complete on android if someone is waiting on the future. we may want to resolve the future immediately to prevent that when not on iOS.